content/articles/ux-colors.md

```diff
@@ -0,0 +1,306 @@
+visual design work in Locus Tempus, a recent web-based application that we developed here at the CTL.
+
+## Logo colors
+
+{{< figure
+    src="/img/assets/ux-colors-lt-logo.svg"
+    alt="Locus Tempus logo, bearing the title, the logo is a map marker in blue, amber and green, with clock face."
+    width="75%"
+    class="mx-auto bordered"
+>}}
+
+Locus Tempus is an open-source digital mapping tool that leverages a geospatial environment to
+engage students as repository builders, researchers and curators. Faculty and students from all
+disciplines can collaboratively identify, organize, and visualize geolocated materials within a
+course context.
+
+I thought it was appropriate to set the tone of Locus Tempus with the color palette in
+[Mapbox](https://www.mapbox.com)—the mapping tool used in Locus Tempus— and other mapping
+applications such as Google Maps, and Apple Maps. These applications have these color palettes in
+common: blue, amber (or yellow), and green. I chose these colors to represent the identity of Locus
+Tempus in its logo.
+
+<div class="row my-5">
+    <div class="col-3 col-md-2 offset-md-1">
+    <img class="mh-100 mw-100" src="/img/assets/ux-colors-lt-avatar.png" />
+    </div>
+    <div class="col-2 p-2 py-3 p-md-3 mr-3" style="line-height: 1; background: #90caf9;">
+    <small>BLUE</small>
+    </div>
+   <div class="col-2 p-2 py-3 p-md-3 mr-3" style="line-height: 1; background: #ffb300;">
+    <small>AMBER</small>
+    </div>
+   <div class="col-2 p-2 py-3 p-md-3 mr-3" style="line-height: 1; color: #fff; background: #218739;">
+    <small>GREEN</small>
+    </div>
+</div>
+
+However, this choice of colors didn’t begin and end in the logo. To reflect the Locus Tempus brand
+throughout the application, I needed to make sure that these colors are in harmony outside their
+use in maps, and examine how they can be used to improve user experience and bolster usability of
+the tool.
+
+## Grayscale matter
+
+At first, I wanted us to focus on the user experience for the tool, so the interface was mostly
+grayscale. When only
+[colors are used as visual cues](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+to convey meaning and guide user behavior, these cues will not be accessible to users who are blind
+or have limited color vision. By leaving the Locus Tempus user interface in grayscale for the most
+part of the design and development phase, we can shape an accessible user interface without color cues.
+
+After we decided on the components, layout, grouping, architecture, and functionalities, and made
+sure the experience is accessible, then I began the process of choosing the right color palette.
+
+
+{{< figure
+    src="/img/assets/ux-colors-lt-bw.png"
+    width="50%"
+    class="mx-auto bordered"
+    alt="Locus Tempus user interface in grayscale."
+>}}
+
+## Colors for user interface
+
+Looking at the palettes in map aforementioned mapping applications, blue is used to represent
```

Comment:
"Looking at the palettes in `the` aforementioned

Reply:
Fixed